app/models/user.rb

```diff
@@ -17,6 +18,8 @@ class User < ActiveRecord::Base
   delegate :can?, :cannot?, to: :ability
   delegate :role_enum, to: :class
 
+  has_many :galleries, inverse_of: :publisher
```

Comment:
I think you need to state the name of the foreign key column here as otherwise this fails: `Gallery.where.not(published_by: nil).first.publisher.galleries`